### ISSUE TITLE
Evaluation interval VS validation interval

### ISSUE THREAD

**Issue by joihn** [open-mmlab/mmdetection#6380]
I am using a **training set**, a **validation set** and a **test set**.

`evaluation = dict(interval=1, metric='bbox')`
This setting means that after each training epoch, mmdet will do an evaluation on the test set, am I correct ?

`workflow = [('train', 1),('val', 1)]`
This setting means that after each training epoch, mmdet will do an evaluation on the validation set, am I correct ?

**Comment by dwluo** [comment]
FYI, according to the info from https://mmdetection.readthedocs.io/en/latest/1_exist_data_model.html, evaluation = dict(interval=1, metric='bbox') performs on validation set.
![image](https://user-images.githubusercontent.com/92134460/208816043-5b938a9a-0ef8-4ff5-acef-3af3f8395feb.png)
